### DISCUSSION TITLE
IWRES and NPDE vs time/concentration plots interpretation

### DISCUSSION THREAD

**Discussion by jmtqt** [Open-Systems-Pharmacology/Forum#1189] [Questions & Problems]
**MODEL 1**
![Model 1](https://user-images.githubusercontent.com/102895649/161534468-a027134f-209a-405e-956d-a99844985e74.png)
**MODEL 2**
![Model 2](https://user-images.githubusercontent.com/102895649/161534501-20048fe0-2354-4589-a540-580e0e2cc5b5.png)
**MODEL 3**
![Model 3](https://user-images.githubusercontent.com/102895649/161534538-f68a9b57-73d5-458f-9511-ce79b2592744.png)

Hello everyone,

I have a question with regards to the scatter plot of residual (especially IWRES and NPDE vs time/concentration). I have two different plots here simulating from two different structural models in NLME PopPk model (see attached Model 1 and Model 2 plots).

My understanding is that the scatter plot of residual tells you how much the predicted data deviates from the observed data. Based on these two plots (IWRES and NPDE of model 1 vs model 2), my understanding is that Model 1 is better than Model 2 only because the interpolation spline of Model 1 (the yellow line) lies on the horizontal y=0 line. In contrast, Model 2 has the trend line scattering unevenly on the horizontal line, and it even increasingly shows deviation when the time and the concentration are higher. Is my interpretation correct?

How about the plot from Model 3 (see attached)? I won't consider this model because all the data skewed towards right, and hence it is telling model misspecification? am I correct in this?

I hope this makes sense.

Thanks in advance

**Comment by PavelBal** [comment]
Can you try to upload the figures directly to GitHub? The links are not working.

**Reply by jmtqt** [reply]
Hi Pavel,

I have uploaded the figures as requested.
